components/eam/bld/namelist_files/use_cases/scam_arm97_MMF-1mom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
+<!-- "IOP: ARM 1997 (18 Jun 1997 - 17 Jul 1997)" -->
+
+<namelist_defaults>
+
+<!-- namelist specific for ARM97 case -->
+<iopfile                                          scam="1"  >atm/cam/scam/iop/ARM97_iopfile_4scam.nc</iopfile>
+<scmlat                                           scam="1"  >   36.6     </scmlat>
+<scmlon                                           scam="1"  >   262.5    </scmlon>
+<start_ymd                                        scam="1"  >   19970619 </start_ymd>
+<start_tod                                        scam="1"  >   84585    </start_tod>
+
+<!-- namelist for single column model -->
+<single_column                                    scam="1"  >  .true.    </single_column>
+<scm_iop_srf_prop                                 scam="1"  >  .true.    </scm_iop_srf_prop>
+<swrad_off                                        scam="1"  >  .false.   </swrad_off>
+<lwrad_off                                        scam="1"  >  .false.   </lwrad_off>
+<precip_off                                       scam="1"  >  .false.   </precip_off>
+<scm_relaxation                                   scam="1"  >  .false.   </scm_relaxation>
+
+<ncdata                                           scam="1"  >atm/cam/inic/homme/cami_mam3_Linoz_ne4np4_L72_c160909.nc</ncdata>
+<soag_ext_file >atm/cam/chem/trop_mozart_aero/emis/aces4bgc_nvsoa_soag_elev_2000_c160427.nc</soag_ext_file>
+<mfilt                                            scam="1"  >   1500     </mfilt>
```

Comment:
Why do you have a history file parameter here? Seems like this shouldn't be part of the use case file.

Reply:
I added it so I can have all history output in a single file. It is easier that way to plot the result. I just copied it from non-MMF SCM case.

Reply:
I think we should let the user decide that rather than bake in a default here.